src/Serilog/Configuration/LoggerSinkConfiguration.cs

```diff
@@ -150,6 +150,29 @@ internal LoggerSinkConfiguration(LoggerConfiguration loggerConfiguration, Action
         {
             if (logger == null) throw new ArgumentNullException(nameof(logger));
             return Sink(new SecondaryLoggerSink(logger, attemptDispose: false), restrictedToMinimumLevel);
-        } 
+        }
+
+        /// <summary>
+        /// Helper method for wrapping sinks.
+        /// </summary>
+        /// <param name="logggerSinkConfiguration">The parent sink configuration.</param>
+        /// <param name="sinkDecorator">A function that allows for wrapping <see cref="ILogEventSink"/>s
```

Comment:
Since it's an action, and the API uses the term _wrap_ rather than _decorate_, could we use `wrapSink`? Also missing a period/full-stop for this sentence.